frontend/src/crosssection/components/CrossMap.vue

```diff
@@ -0,0 +1,199 @@
+    this.map = null
+  },
+  methods: {
+    initMapBox () {
+      if (!mapboxgl.supported()) {
+        this.browserUnsupported = true
+        return
+      }
+
+      var mapConfig = {
+        container: this.$el,
+        zoom: 4,
+        minZoom: 4,
+        maxPitch: 0,
+        dragRotate: false,
+        center: CENTRE_LNG_LAT_BC,
+        style: {
+          version: 8,
+          sources: {
+            [DATABC_ROADS_SOURCE_ID]: DATABC_ROADS_SOURCE,
+            [DATABC_CADASTREL_SOURCE_ID]: DATABC_CADASTREL_SOURCE,
+            [WELLS_SOURCE_ID]: WELLS_SOURCE,
+            [FOCUSED_WELLS_SOURCE_ID]: { type: 'geojson', data: buildWellsGeoJSON([]) }
+          },
+          layers: [
+            DATABC_ROADS_LAYER,
+            DATABC_CADASTREL_LAYER,
+            //should we filter based on user role? this page is already behind keycloak
+            wellsBaseAndArtesianLayer({ filter: wellLayerFilter(true) }),
+            focusedWellsLayer()
+          ]
+        }
+      }
+
+      this.map = new mapboxgl.Map(mapConfig)
+      new GestureHandling({ modifierKey: 'ctrl' }).addTo(this.map)
+
+      /* Add controls */
+
+      this.map.addControl(new mapboxgl.NavigationControl({ showCompass: false }), 'top-left')
+      this.map.addControl(new mapboxgl.GeolocateControl({
+        positionOptions: {
+          enableHighAccuracy: true
+        }
+      }), 'top-left')
+      this.map.addControl(new mapboxgl.ScaleControl({
+        maxWidth: 80,
+        unit: 'imperial'
+      }))
+      this.map.addControl(new mapboxgl.ScaleControl({
+        maxWidth: 80,
+        unit: 'metric'
+      }))
+      this.map.addControl(new mapboxgl.AttributionControl({
+        customAttribution: 'MapBox | Government of British Columbia, DataBC, GeoBC '
+      }))
+
+      this.legendControl = new LegendControl({
+        layers: this.mapLayers
+      })
+      this.map.addControl(this.legendControl, 'bottom-right')
+
+      this.map.on('load', () => {
+        this.map.addImage(FOCUSED_WELL_ARTESIAN_IMAGE_ID, new PulsingArtesianWellImage(this.map), { pixelRatio: 2 })
+        this.map.addImage(FOCUSED_WELL_IMAGE_ID, new PulsingWellImage(this.map), { pixelRatio: 2 })
+        this.map.addImage(FOCUSED_WELL_CLOSED_IMAGE_ID, new PulsingClosedWellImage(this.map), { pixelRatio: 2 })
+
+        const tooltipLayers = {
+          [WELLS_BASE_AND_ARTESIAN_LAYER_ID]: {
+            snapToCenter: true,
+            createTooltipContent: this.createWellPopupElement
+          }
+        }
+
+        setupFeatureTooltips(this.map, tooltipLayers)
+
+        this.setFocusedWells(this.focusedWells)
+
+        this.$emit('mapLoaded')
+      })
+    },
+    setFocusedWells (wells) {
+      this.map.getSource(FOCUSED_WELLS_SOURCE_ID).setData(buildWellsGeoJSON(wells, FOCUSED_WELL_PROPERTIES))
+    },
+    createWellPopupElement (features, { canInteract }) {
+      return createWellPopupElement(features, this.map, this.$router, {
+        canInteract,
```

Comment:
You should include the new variable I made, `openInNewTab`. It's merged to main now.